spec/string_calculator_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "string_calculator"
+
+describe StringCalculator do
+
+  describe ".add" do
+    context "given an empty string" do
+      it "returns zero" do
+        expect(StringCalculator.add("")).to eql(0)
```

Comment:
You could use described_class https://www.relishapp.com/rspec/rspec-core/docs/metadata/described-class - it would help once you decide to change the class's name.